middleware_and_drivers.md

```diff
@@ -0,0 +1,17 @@
+The aim for this page is to provide a collection of ROS packages/related libraries which are very useful for aerial ROS projects either to serve as a reference or directly use as a part of a custom software stack. This list aims to address the gap between the flight controller firmware and purely autonomy related packages. 
```

Comment:
Could you add `#Middelware and Drivers` as title on top here?

Reply:
Sure, done.